### ISSUE TITLE
Button to choose video quality resolution and scroll wheel button increase or decrease sound volume and button "audio only" for online youtube video streams

### ISSUE THREAD

**Issue by trimechee** [vzhd1701/gridplayer#56]
### Suggestion

Hello, it will be great to add these option please tant make onr lives more confortable please :

youtube has dash format for video and has audio format like opus and m4a, , it will be great to add button "audio only" for online youtube video streams therefore we can save data , and add quick  button on the main window to choose video quality resolution and make scroll wheel button increase or decrease sound volume, thank you very much !

does Girdplayer update automatically yt-dlp and streamlink to last version when start ?

**Comment by vzhd1701** [comment]
> does Girdplayer update automatically yt-dlp and streamlink to last version when start ?

No, but I will try to keep up to date with them and release a new version of GridPlayer when yt-dlp or streamlink updates

**Comment by trimechee** [comment]
Great !! Thank you very much :)
